crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -133,6 +132,34 @@ pub(crate) trait StateReadExt: StateRead + address::StateReadExt {
             .wrap_err("invalid bridge account withdrawer address bytes")
     }
 
+    #[instrument(
+        skip_all,
+        fields(address = %address.display_address(), withdrawal_event_id),
+        err(level = Level::DEBUG)
+    )]
+    async fn get_withdrawal_event_block_for_bridge_account<T: AddressBytes>(
```

Comment:
```suggestion
    async fn get_withdrawal_event_block_height_for_bridge_account<T: AddressBytes>(
```

Reply:
actually, this is using the rollup block number as a way to ensure withdrawals aren't double-executed, so i'd change this to `get_withdrawal_event_rollup_block_number` or something like that, or add a comment to make the reasoning for this clearer